### DISCUSSION TITLE
Cannot get temp to display

### DISCUSSION THREAD

**Discussion by jkmhb** [tagyoureit/nodejs-poolController#602] [Q&A]
![image](https://user-images.githubusercontent.com/50624053/189498638-9c736793-f419-4990-ae69-9e482d21200b.jpeg)![image](https://user-images.githubusercontent.com/50624053/189498649-0325596c-1f48-4ab1-b51f-19c0be8090cf.jpeg)Hi, I am using sequent building automation hat. I've got resistance to display, what am I missing?![image](https://user-images.githubusercontent.com/50624053/189498663-4c87f8b7-f0ec-4483-a68f-7dcbe0c8e4fa.jpeg)![image](https://user-images.githubusercontent.com/50624053/189498668-6aebef0a-5bbf-4332-9ec1-e1a871b6cbd8.jpeg)![image](https://user-images.githubusercontent.com/50624053/189498676-dc1133c0-a52a-48ab-8fe2-92e8588fcdb5.jpeg)

**Comment by tagyoureit** [comment]
It looks like you have resistance on inputs 1 and 2 but have feeds from 1 and 3. You should be feeding voltage to the adcValue. Make sure your feeds look like these:  https://github.com/rstrouse/relayEquipmentManager/wiki/Temperature-Sensors#step-5-feed-your-temperature-probe

**Reply by rstrouse** [reply]
Actually not voltage since it’s output is in kohms on set your feed up as described here.  Make sure you select the output labeled value for the feed to the probe.  https://github.com/rstrouse/relayEquipmentManager/wiki/Sequent-Microsystems-Boards#inputs-tab

Then set the input type to Resistance on the probe and the units to kohm.

**Comment by jkmhb** [comment]
I guess I should have been more clear. I have a temp sensor in position 1 and 2, and a pressure sensor (0-5v) in position 3. I thought if I could get the 1st temp sensor (air temp) to work, I could figure out the others.

**Comment by jkmhb** [comment]
thank you!!!
